Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -185,6 +218,20 @@ - (void)handleRegisterUserNotificationSettings:(NSNotification *)notification
   _requestPermissionsResolveBlock = nil;
 }
 
+RCT_EXPORT_METHOD(onFinishRemoteNotification:(NSString*)notificationId fetchResult:(UIBackgroundFetchResult)result) {
+  RemoteNotificationCallback completionHandler = self.remoteNotificationCallbacks[notificationId];
+  if (completionHandler == nil) {
+    @throw [NSException
```

Comment:
This doesn't seem like it should be a fatal error. Instead use RCTLogError (it will show a redbox too)